frontend/mulighetsrommet-cms/schemas/faneinnhold.ts

```diff
@@ -123,9 +127,17 @@ export const faneinnhold = defineType({
           blockContentValidation(
             doc,
             MAKS_LENGDE_INNHOLD,
-            "Innholdet er for langt. Kan innholdet være kortere og mer konsist?"
-          )
+            "Innholdet er for langt. Kan innholdet være kortere og mer konsist?",
+          ),
         ),
     }),
+    defineField({
+      name: "lenker",
+      title: "Lenker",
+      description: "KOMMER SNART - Legg til lenker som blir synlig for NAV-ansatte",
+      type: "array",
+      of: [{ type: "lenke" }],
+      readOnly: true,
```

Comment:
read-only fordi jeg ikke vil lage feature toggle for dette. Så lenge redaktør ikke kan legge inn lenker er det ingenting som vises i frontend til veiledere.